src/platform/user/authentication/components/LoginActions.jsx

```diff
@@ -34,6 +42,7 @@ export default function LoginActions({ externalApplication }) {
 
   return (
     <div className="row">
+      {showsignInPageAndModalExperiment && <SignInAlertBox />}
```

Comment:
`externalApplication` is the prop you want to check for all the various USiP applications.

1. If `externalApplication` is null, we treat it as the default (or normal VA.gov login)
2. The list of `externalApplications` can be found in the `user/authentication/constants.js` file